backend/app/controllers/enrollments_controller.rb

```diff
@@ -1,4 +1,6 @@
 class EnrollmentsController < ApplicationController
+  include ActionController::MimeResponds
```

Comment:
You don't actually need. This module includes `respond_to` method, which is convenient when you have to deal with multiple content type, which is not the case here.